src/Catalyst.Core.Modules.Web3/Controllers/EthController.cs

```diff
@@ -47,9 +49,33 @@ public EthController(IWeb3EthApi web3EthApi, IWeb3HandlerResolver handlerResolve
             _handlerResolver = handlerResolver ?? throw new ArgumentNullException(nameof(handlerResolver));
             _jsonSerializer = jsonSerializer ?? throw new ArgumentNullException(nameof(jsonSerializer));
         }
+        
+        private JsonRpcResponse[] HandleManyRequests(JsonRpcRequest[] requests)
+        {
+            JsonRpcResponse[] responses = new JsonRpcResponse[requests.Length];
+            for (int i = 0; i < requests.Length; i++)
+            {
+                responses[i] = HandleSingleRequest(requests[i]);
+            }
+
+            return responses;
+        }
 
         [HttpPost]
-        public JsonRpcResponse Request([FromBody] JsonRpcRequest request)
+        public object Request([FromBody] object body)
+        {
+            JObject jObject = body as JObject;
+            if (jObject?.Type == JTokenType.Array)
+            {
+                JsonRpcRequest[] requests = jObject.ToObject<JsonRpcRequest[]>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jObject' is null on at least one execution path.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=5051857)